src/generator-utilities.js

```diff
@@ -120,6 +86,10 @@ function getReactComponentDirs(name, directory, native, redux) {
             template: path.join(templateDirectory, 'test', 'template.container.spec.js'),
             generated: path.join(generatedDirectory, 'test', `${name}.container.spec.js`)
         },
+        action: {
```

Comment:
@YashKalaria-BTC It seems we removed the `createAction` JSON object and added this to simplify, I would like to understand this change.